Week1/homework/app.js

```diff
@@ -1,11 +1,158 @@
-'use strict';
-
 {
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
-  ];
+  // 1.1
+
+  // const myFavoriteBooks = [
+  //   '1984',
+  //   'white_fang',
+  //   'iron_heel',
+  //   'animal_farm',
+  //   'les_miserables',
+  //   'war_and_peace',
+  //   'crime_and_punishment',
+  //   'the_great_gasby',
+  //   'the_red_and_the_black',
+  //   'the_lord_of_the_rings',
+  // ];
+
+  // 1.3
+
+  // const listBooks = () => {
```

Comment:
Just a suggestion - a better function name could be `generateListOfBooks`. If you look at the Naming Conventions document from JS1, function names should contain a verb which describes the action being performed by it which in this case, we are trying to generate a list of books.